### PR TITLE
Spelling: Marked nouns, whitespace, upgrade, : Bigversal

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by micah** [comment]
Overall, I think that these are good changes that we should accept! I'm not sure about changing the 'must' to a 'can' in that one section, but the others I think are good!

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -3,41 +3,41 @@
...
+    <string name="translation_project_text">Translations welcome and appreciated. See our Transifex project at https://www.transifex.com/projects/p/bitmask-android/</string>
     <string name="switch_provider_menu_option">Switch provider</string>
     <string name="info">info</string>
     <string name="show_connection_details">Show connection details</string>
     <string name="routes_info">Routes: %s</string>
-    <string name="routes_info6">Routes IPv6: %s</string>
-    <string name="error_empty_username">The username must not be empty.</string>
-    <string name="cert_from_keystore">Got certificate \'%s\' from Keystore</string>
+    <string name="routes_info6">IPv6 routes: %s</string>
+    <string name="error_empty_username">The username can not be empty.</string>
```
Comment (cyBerta) [inline app/src/main/res/values/strings.xml]: could you revert this change? Besides that: great!

**Comment by cyBerta** [comment]
Thanks for your contribution @comradekingu!

**Comment by cyBerta** [comment]
could you please set the target branch to 0.9.8? That's our current kind-of-development-branch.

**Comment by cyBerta** [comment]
Stupid merge fuck up... anyways - I've merged your commit manually.
